test/sass4clj/core_test.clj

```diff
@@ -3,29 +3,8 @@
             [sass4clj.core :refer :all])
   (:import [java.io File]))
 
-(def sass
```

Comment:
> It also seemed like they created ambiguity that runs counter to how Sass is used. @import is supposed to take a relative path that resolves to a single absolute path, but these tests were treating the import paths as relative to both the temporary file and the resources directory.

I copied the logic used by the less4clj so it is possible that it doesn't completely make sense here.